### PR TITLE
Add tutorials link to main doc page

### PR DESCRIPTION
**Pull request by mirca**
cc @eteq @bsipocz @eblur @kelle

### CODE REVIEW

**Comment by bsipocz** [comment]
Hey, actually someone did something rather just talking. Well done @mirca.

File: docs/index.rst [hunk 1]
```diff
@@ -59,6 +59,7 @@ User Documentation
    install
    getting_started
    Example gallery <generated/examples/index>
+   Tutorials <http://tutorials.astropy.org/>
```
Comment (pllim) [inline docs/index.rst]: I don't think this is the correct formatting though.
```
`Tutorials <http://tutorials.astropy.org/>`_
```

**Comment by mirca** [comment]
Looks fine here (and the link works) @pllim  screen shot 2017-05-08 at 4 25 16 pm

File: docs/index.rst (hunk 1 above)
Comment (bsipocz) [inline docs/index.rst]: It's working like this, your suggestion is needed when you want to refer to the link as `Tutorial`_ later.

**Comment by bsipocz** [comment]
Hmm, should have looked at the new comments first here rather than build the docs locally....

**Comment by mirca** [comment]
@bsipocz a tool to just test/build part of the sphinx documentation would be very useful, wouldn't it?

**Comment by bsipocz** [comment]
Sure. Am I hearing it right that you just volunteered to dig up something for that? (though I think it wouldn't have worked here as this docs page doesn't live in one of the submodules).

**Pull request merged**